backend/e4h-services/im-services/src/main/java/org/egov/im/web/models/UserRequest.java

```diff
@@ -0,0 +1,26 @@
+package org.egov.im.web.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+import org.egov.common.contract.request.RequestInfo;
+
+import javax.validation.constraints.NotNull;
+
+@AllArgsConstructor
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Setter
+@ToString
+@Builder
+public class UserRequest {
+
+	@NotNull
+	@JsonProperty("RequestInfo")
+	private RequestInfo requestInfo;
+
+	@NotNull
+	@JsonProperty("User")
+	private User user;
```

Comment:
Prefer using the eGov common user contract wherever applicable. Avoid duplicating common contracts unless the existing one does not meet your specific requirements.